install/kubernetes/helm/istio/charts/tracing/templates/deployment.yaml

```diff
@@ -1,34 +1,35 @@
-{{ if eq .Values.provider "jaeger" }}
-
 apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: istio-tracing
   namespace: {{ .Release.Namespace }}
   labels:
-    app: jaeger
+    app: {{ template "tracing.name" . }}
     chart: {{ template "tracing.chart" . }}
     heritage: {{ .Release.Service }}
     release: {{ .Release.Name }}
 spec:
   selector:
     matchLabels:
-      app: jaeger
+      app: {{ template "tracing.name" . }}
   template:
     metadata:
       labels:
-        app: jaeger
+        app: {{ template "tracing.name" . }}
         chart: {{ template "tracing.chart" . }}
         heritage: {{ .Release.Service }}
         release: {{ .Release.Name }}
       annotations:
         sidecar.istio.io/inject: "false"
+        scheduler.alpha.kubernetes.io/critical-pod: ""
```

Comment:
Please remove the critical-pod annotation as it's going to be deprecated and removed in future k8s version, see: https://kubernetes.io/docs/tasks/administer-cluster/guaranteed-scheduling-critical-addon-pods/#marking-pod-as-critical